diff --git a/matplotlibrc.template b/matplotlibrc.template
index 2f0e5c4..c9b6e49 100644
--- a/matplotlibrc.template
+++ b/matplotlibrc.template
@@ -38,12 +38,12 @@
 #
 # If you omit this parameter, it will always default to "Agg", which is a
 # non-interactive backend.
-backend      : $TEMPLATE_BACKEND
+backend      : Qt4Agg
 
 # If you are using the Qt4Agg backend, you can choose here
 # to use the PyQt4 bindings or the newer PySide bindings to
 # the underlying Qt4 toolkit.
-#backend.qt4 : PyQt4        # PyQt4 | PySide
+backend.qt4 : PyQt4        # PyQt4 | PySide
 
 # Note that this can be overridden by the environment variable
 # QT_API used by Enthought Tool Suite (ETS); valid values are
