diff --git setupext.py setupext.py
index 0ae553a..66c6e02 100755
--- setupext.py
+++ setupext.py
@@ -1425,6 +1425,7 @@ class BackendTkAgg(OptionalBackendPackage):
             tk.destroy()
 
         # Save directories and version string to cache
+#       tcl_lib_dir = tk_lib_dir = r'C:\Tcl%d' % (8 * tuple.__itemsize__)
         self.tcl_tk_cache = tcl_lib_dir, tk_lib_dir, str(Tkinter.TkVersion)[:3]
         return self.tcl_tk_cache
 
@@ -1531,13 +1532,10 @@ class BackendTkAgg(OptionalBackendPackage):
     def add_flags(self, ext):
         if sys.platform == 'win32':
             major, minor1, minor2, s, tmp = sys.version_info
-            if sys.version_info[0:2] < (3, 4):
-                ext.include_dirs.extend(['win32_static/include/tcl85'])
-                ext.libraries.extend(['tk85', 'tcl85'])
-            else:
-                ext.include_dirs.extend(['win32_static/include/tcl86'])
-                ext.libraries.extend(['tk86t', 'tcl86t'])
-            ext.library_dirs.extend([os.path.join(sys.prefix, 'dlls')])
+            tcl_dir = r'C:\Tcl%d' % (8 * tuple.__itemsize__)
+            ext.include_dirs.append(os.path.join(tcl_dir, 'include'))
+            ext.library_dirs.append(os.path.join(tcl_dir, 'lib'))
+            ext.libraries.extend(['tk85', 'tcl85'])
 
         elif sys.platform == 'darwin':
             # this config section lifted directly from Imaging - thanks to
